### PR TITLE
Update file-storage.md

### PR DESCRIPTION
**Pull request by TeresaFG-writer**
Added content initially provided by Madhuri and edited it, but now need to go back in and add a hyperlink [Azure Active Directory B2B Documentation(https://docs.microsoft.com/en-us/azure/active-directory/b2b/).

### CODE REVIEW

**Comment by TeresaFG-writer** [comment]
@matondep Have a look.

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: yammer/get-started-with-yammer/file-storage.md [hunk 1]
```diff
@@ -89,16 +89,16 @@ Files will continue to be stored in Yammer cloud storage in the following instan
...
+|**Azure B2B guest, but not a member of a the group<br/>Network-level guest<br/>Conversation-level guest that is not in your network**|Automatic file access is not allowed. These users can request access to specific files.<br/>Can't upload files.|Automatic file access is not permitted. Guest users can request access to specific files. File upload is not permitted.|   
+|**Network-level guest, but not Azure B2B guest**|Automatic file access is not allowed. A guest user must become an Azure B2B guest and a member of the group in Office 365. Alternatively, other group members can grant access to specific files or the entire document library through one of many SharePoint external sharing methods.|No automatic access for network level guests to Yammer files saved in SharePoint. If you want to enable access to specific files, add them as an Azure B2B guest on the Office 365 tenant. For more information, see [Azure Active Directory B2B Documentation] (https://docs.microsoft.com/en-us/azure/active-directory/b2b/). If guest users need to upload files to a specific group from SharePoint or have automatic access to files uploaded to SharePoint, add them as a group member in SharePoint.
```
Comment (JohanFreelancer9) [inline yammer/get-started-with-yammer/file-storage.md]:
```suggestion
|**Network-level guest, but not Azure B2B guest**|Automatic file access is not allowed. A guest user must become an Azure B2B guest and a member of the group in Office 365. Alternatively, other group members can grant access to specific files or the entire document library through one of many SharePoint external sharing methods.|No automatic access for network level guests to Yammer files saved in SharePoint. If you want to enable access to specific files, add them as an Azure B2B guest on the Office 365 tenant. For more information, see [Azure Active Directory B2B documentation] (https://docs.microsoft.com/azure/active-directory/b2b/). If guest users need to upload files to a specific group from SharePoint or have automatic access to files uploaded to SharePoint, add them as a group member in SharePoint.
```

**Comment by mypil** [comment]
@TeresaFG-writer -  Please take note that the table does not display correctly. Also, kindly merge the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. 

Thank you.

File: yammer/get-started-with-yammer/file-storage.md [hunk 2]
```diff
@@ -89,16 +89,16 @@ Files will continue to be stored in Yammer cloud storage in the following instan
...
 The following table shows how each type of guest and external user can access files uploaded in Yammer and stored in SharePoint.
 
 |||
-|----------|----------|
-|**Type of user**|**Access to group files**|
-|**Conversation-level guest that is in your network**|**Private group**: can view files that have been shared in the conversation, but can't upload files.<br>**Public group**: Can view, edit, and upload files.|
-|**Network-level guest that is also an Azure B2B guest, and also a member of the group in Office 365**|Can view, edit and upload files.|
-|**Azure B2B guest, but not a member of a the group<br/>Network-level guest<br/>Conversation-level guest that is not in your network**|No file access by default. These users can request access to specific files.<br/>Can't upload files.|
-|**Network-level guest, but not Azure B2B guest**| No file access. These users must become an Azure B2B guest and a member of the group in Office 365. Alternatively, other group members can grant access to specific files or the entire document library via one of many SharePoint external sharing methods.|
+|----------|----------|----------|
```
Comment (nenonix) [inline yammer/get-started-with-yammer/file-storage.md]:
```suggestion
|**Type of user**|**Access to group files in Yammer**|**Access to group files in SharePoint**|
```

File: yammer/get-started-with-yammer/file-storage.md [hunk 3]
```diff
@@ -89,16 +89,16 @@ Files will continue to be stored in Yammer cloud storage in the following instan
...
+|**Type of user**|**Access to group files in Yammer**|**Access to group files in SharePoint**|
```
Comment (nenonix) [inline yammer/get-started-with-yammer/file-storage.md]:
```suggestion
|----------|----------|----------|
```

File: yammer/get-started-with-yammer/file-storage.md [hunk 4]
```diff
@@ -89,16 +89,16 @@ Files will continue to be stored in Yammer cloud storage in the following instan
...
+|**Azure B2B guest, but not a member of a the group<br/>Network-level guest<br/>Conversation-level guest that is not in your network**|Automatic file access is not allowed. These users can request access to specific files.<br/>Can't upload files.|Automatic file access is not permitted. Guest users can request access to specific files. File upload is not permitted.|   
+|**Network-level guest, but not Azure B2B guest**|Automatic file access is not allowed. A guest user must become an Azure B2B guest and a member of the group in Office 365. Alternatively, other group members can grant access to specific files or the entire document library through one of many SharePoint external sharing methods.|No automatic access for network level guests to Yammer files saved in SharePoint. If you want to enable access to specific files, add them as an Azure B2B guest on the Office 365 tenant. For more information, see [Azure Active Directory B2B documentation] (https://docs.microsoft.com/azure/active-directory/b2b/). If guest users need to upload files to a specific group from SharePoint or have automatic access to files uploaded to SharePoint, add them as a group member in SharePoint.
```
Comment (nenonix) [inline yammer/get-started-with-yammer/file-storage.md]:
```suggestion
|**Network-level guest, but not Azure B2B guest**|Automatic file access is not allowed. A guest user must become an Azure B2B guest and a member of the group in Office 365. Alternatively, other group members can grant access to specific files or the entire document library through one of many SharePoint external sharing methods.|No automatic access for network level guests to Yammer files saved in SharePoint. If you want to enable access to specific files, add them as an Azure B2B guest on the Office 365 tenant. For more information, see [Azure Active Directory B2B documentation](https://docs.microsoft.com/azure/active-directory/b2b/). If guest users need to upload files to a specific group from SharePoint or have automatic access to files uploaded to SharePoint, add them as a group member in SharePoint.
```

Comment (nenonix) [reply yammer/get-started-with-yammer/file-storage.md]: Note the same link is provided below the table in the source content.

File: yammer/get-started-with-yammer/file-storage.md [hunk 5]
```diff
@@ -89,16 +89,16 @@ Files will continue to be stored in Yammer cloud storage in the following instan
...
 |||
```
Comment (nenonix) [inline yammer/get-started-with-yammer/file-storage.md]:
```suggestion
```

File: yammer/get-started-with-yammer/file-storage.md [hunk 6]
```diff
@@ -89,16 +89,16 @@ Files will continue to be stored in Yammer cloud storage in the following instan
 Files that users upload in Office 365 connected groups are saved in the **Apps > Yammer** subfolder of the SharePoint document library for the Office 365 connected group. The SharePoint document library can be accessed from Yammer under **Office 365 Resources** on the right side of an Office 365 connected Yammer group, as well as through SharePoint itself.   
   
 ## Guest and external user access to files
-  
+
 The following table shows how each type of guest and external user can access files uploaded in Yammer and stored in SharePoint.
 
 |||
```
Comment (nenonix) [inline yammer/get-started-with-yammer/file-storage.md]:
```suggestion
```

**Comment by mypil** [comment]
@TeresaFG-writer - Kindly merge the additional commit made by nenonix for the copy/edit review and to correct the table. Thank you.

**Comment by mypil** [comment]
@IrenePasternack - PR has been copy edited and is ready for final review.

Thank you.

**Comment by CrystalThomasMS** [comment]
I'm working on finding out who the new writer is responsible for Yammer admin content. Thanks for your patience!

**Comment by JoanneHendrickson** [comment]
Teresa is the new writer for Yammer content.  We are sorting out some issues with her perms

**Comment by mypil** [comment]
@JoanneHendrickson - May we know if the manager for the Yammer content is still pamgreen?
Please advise so we can update the metadata author/manager.

cc: @kenwith

**Pull request merged**